### PR TITLE
Yarn information for the GSG docs

### LINKED ISSUE #4841: Add a trouble shooting section for yarn
At least specify the minimal version to be used.

Also give a hint regarding
> (Ubuntu 17.04 comes with cmdtest installed by default.
> If you’re getting errors from installing yarn, you may want to run sudo apt remove cmdtest first.)

### PR DESCRIPTION
**Pull request by nemanja-da**
Adding information about minimal yarn version and how to troubleshoot it for Debain/Ubuntu resolves #4841

CHANGELOG_BEGIN
CHANGELOG_END

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

**Comment by cocreature** [comment]
I don’t think this should go in the changelog.

**Comment by nemanja-da** [comment]
Removed it

File: docs/source/getting-started/index.rst [hunk 1]
```diff
@@ -78,7 +81,7 @@ In one terminal, at the root of the ``create-daml-app`` directory, run the scrip
 
     ./daml-start.sh
 
-This script is just shorthand for ``daml start`` with some arguments, which does a few things:
+You will know that script has run successfully when you see the ``INFO  com.digitalasset.http.Main$ - Started server: ServerBinding(/0:0:0:0:0:0:0:0:7575)`` message in terminal. The script is just shorthand for ``daml start`` with some arguments, which does a few things:
```
Comment (cocreature) [inline docs/source/getting-started/index.rst]: Is  `0.0.0.0` still accurate? I thought we changed that to `127.0.0.1`.

Comment (nemanja-da) [reply docs/source/getting-started/index.rst]: That's copy/pasted from terminal 🤷‍♂

Comment (cocreature) [reply docs/source/getting-started/index.rst]: on which release? On the latest snapshot I get `Started server: ServerBinding(/127.0.0.1:7575)`

Comment (nemanja-da) [reply docs/source/getting-started/index.rst]: 0.13.54  (project SDK version from daml.yaml)

Comment (cocreature) [reply docs/source/getting-started/index.rst]: Hm, I don’t quite know what the current plans are for keeping create-daml-app in sync with this repo. It seems very weird to have the docs for create-daml-app which are published as part of the SDK docs document an older version. Either way, happy to keep it like it is for now but maybe add a comment so we don’t forget to update it.

cc @hurryabit @rohanjr

Comment (nemanja-da) [reply docs/source/getting-started/index.rst]:
```suggestion
You will know that script has run successfully when you see the ``INFO  com.digitalasset.http.Main$ - Started server: ServerBinding(/0:0:0:0:0:0:0:0:7575)`` or ``INFO  com.digitalasset.http.Main$ - Started server: ServerBinding(/127.0.0.1:7575)`` message in terminal. The script is just shorthand for ``daml start`` with some arguments, which does a few things:
```

@cocreature would this be better?

**Pull request merged**